javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,25 @@
+// This code will move the li to the position 100, 100. Calling moveElement will return a promise that resolves after the li element has been moved.
+
+function translateOneByOne(){
+    moveElement(document.querySelector("ul.marks li:nth-child(1)"), { x: 20, y: 300 }).then(() => {
```

Comment:
You can use `redBox, blueBox, greenBox` variables that already declared in `move-element.js` file

Reply:
sure I  will,THankyou for your feedback :)